substrate/frame/message-queue/src/lib.rs

```diff
@@ -525,12 +525,21 @@ pub mod pallet {
 		type MaxStale: Get<u32>;
 
 		/// The amount of weight (if any) which should be provided to the message queue for
-		/// servicing enqueued items.
+		/// servicing enqueued items `on_initialize`.
 		///
 		/// This may be legitimately `None` in the case that you will call
 		/// `ServiceQueues::service_queues` manually.
```

Comment:
```suggestion
		/// This may be legitimately `None` in the case that you will call
		/// `ServiceQueues::service_queues` manually or set 
		/// [`Self::IdleMaxServiceWeight`] to have it run in `on_idle`.
```